src/app/components/PriceChart.tsx

```diff
@@ -4,30 +4,51 @@ import {
   CANDLE_PERIODS,
   OHLCVData,
   setCandlePeriod,
+  handleCrosshairMove,
+  // fetchCandlesForInitialPeriod,
+  initializeLegend,
 } from "../redux/priceChartSlice";
 import { useAppDispatch, useAppSelector } from "../hooks";
+import { formatPercentageChange } from "../utils";
 
 interface PriceChartProps {
   data: OHLCVData[];
 }
 
 function PriceChartCanvas(props: PriceChartProps) {
   const chartContainerRef = useRef<HTMLDivElement>(null);
+  const dispatch = useAppDispatch();
   const { data } = props;
+
   useEffect(() => {
     const chartContainer = chartContainerRef.current;
 
+    // dispatch(fetchCandlesForInitialPeriod());
+    if (data && data.length > 0) {
+      dispatch(initializeLegend());
+    }
+
     if (chartContainer) {
       const handleResize = () => {
         chart.applyOptions({ width: chartContainer.clientWidth });
       };
 
       const chart = createChart(chartContainer, {
         width: chartContainer.clientWidth,
-        height: 450,
-
-        // TODO: timescale is not visible
+        height: 500,
+        //MODIFY THEME COLOR HERE
+        layout: {
+          background: { color: "#181c27" },
+          textColor: "#DDD",
+        },
+        //MODIFY THEME COLOR HERE
+        grid: {
+          vertLines: { color: "#444" },
+          horzLines: { color: "#444" },
+        },
         timeScale: {
+          //MODIFY THEME COLOR HERE
+          borderColor: "#d3d3d4",
```

Comment:
These colors probably can also come from the theme settings. You can find what colors we currently have at `tailwind.config.js` under `daisyui` objects (only dark theme currently).

Here is the list of color names and variables that you can use - https://daisyui.com/docs/colors/#-2 . Here in the chart settings you can use them as variables (e.g. `hsl(var(--b1))`). In other places where you style an html element in .tsx you can specify them as classes, e.g. for background `bg-base-100`  or for text color `text-base-100`.